helm_deploy/hmpps-book-a-video-link-api/values.yaml

```diff
@@ -39,6 +39,7 @@ generic-service:
     APPLICATIONINSIGHTS_CONFIGURATION_FILE: applicationinsights.json
     DB_SSL_MODE: "verify-full"
     HMPPS_SQS_USE_WEB_TOKEN: true
+    FEATURE_PROBATION_LIVE: false
```

Comment:
Turned off in dev by default.  Want to make sure existing journey works before turning on.